Develappers.BillomatNet/Api/Json/CollectionConverter.cs

```diff
@@ -7,11 +7,17 @@ namespace Develappers.BillomatNet.Api.Json
 {
     internal class CollectionConverter<T> : JsonConverter
     {
+        /// <summary>
+        /// Checks if the object is a List
```

Comment:
besser: "checks whether this item can be converted"
und bitte "list" und "type" klein schreiben im englischen